data/CHAIN/data.json

```diff
@@ -16,7 +16,7 @@
       }
     },
     "base-goerli": {
-      "address": "0x0c79e0E0284a631b72946b27d8F61EA6d9bF45c6"
+      "address": "0x07C69dFF64b2eC3f1DC45Cd829216c78e7Fe7662"
```

Comment:
this address looks wrong (it's not pointing to a contract): https://basescan.org/address/0x07C69dFF64b2eC3f1DC45Cd829216c78e7Fe7662

Reply:
It's a valid contract address: https://goerli.basescan.org/address/0x07C69dFF64b2eC3f1DC45Cd829216c78e7Fe7662

Reply:
Updated contract deployed at: https://goerli.basescan.org/address/0xeEFf5B1cd3DE16F347996725F5184fdAA54565A0